### DISCUSSION TITLE
Testers wanted for experimental application bundles

### DISCUSSION THREAD

**Discussion by probonopd** [helloSystem/hello#227] [General]
On https://github.com/helloSystem/Applications/releases/ I have started to publish some experiemental application bundles.

These get generated in a GitHub Action using a [simple shell script](https://github.com/helloSystem/Applications/blob/main/makebundle) at the moment. Maybe Python should be used in the future.

The advantage might be that we can up/down/crossgrade applications somewhat independent of the operating system. For example we could produce application bundles from `latest` while the operating system stays on `release_X` or `quarterly` packages.

Known issues:
* Must use a helloSystem 0.7.0 prerelease build
* Must unzip the zip because mounting large ones can fail (archivemount bug?)

**What do people think about this concept?**

**Comment by grahamperrin** [comment]


**Reply by probonopd** [reply]
No, I menat `latest` packages.
